source/integrate-with-integration-environment/process-identity-information.html.md.erb

```diff
@@ -213,46 +213,46 @@ Each JSON object in the list may contain any of the following properties:
   </tr>
   <tr>
     <td class="tg-ktyi"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent"><code>organisationName</code></span></td>
-    <td class="tg-ktyi"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">Maps to <code>ORGANISATION_NAME</code> in the </span><a href="https://www.royalmail.com/find-a-postcode"><span>Postcode Address File</span></a><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent"> and </span><a href="https://apidocs.os.uk/docs/os-places-dpa-output"><span>Ordnance Survey Places API</span></a><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">. </span><br></td>
+    <td class="tg-ktyi"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">Maps to <code>ORGANISATION_NAME</code> in the </span><a href=https://www.royalmail.com/find-a-postcode><span>Postcode Address File</span></a><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent"> and </span><a href="https://apidocs.os.uk/docs/os-places-dpa-output"><span>Ordnance Survey Places API</span></a><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">. </span><br></td>
```

Comment:
While we're here, what are the extra `span` elements for that have neither class nor style? :-)

Reply:
no - they were just autofilled when I converted the table from Markdown > HTML @sabrinaharris